wrappers/golang/core/msm.go

```diff
@@ -32,19 +32,19 @@ type MSMConfig struct {
 	/// Can be set to 0 to disable separate treatment of large buckets altogether. Default value: 10.
 	LargeBucketFactor int32
 
-	batchSize int32
+	BatchSize int32
 
-	areScalarsOnDevice bool
+	AreScalarsOnDevice bool
 
 	/// True if scalars are in Montgomery form and false otherwise. Default value: true.
 	AreScalarsMontgomeryForm bool
 
-	arePointsOnDevice bool
+	ArePointsOnDevice bool
 
 	/// True if coordinates of points are in Montgomery form and false otherwise. Default value: true.
 	ArePointsMontgomeryForm bool
 
-	areResultsOnDevice bool
+	AreResultsOnDevice bool
```

Comment:
These should remain private as the `XXXCheck` functions should manipulate them internally based on the slice arguments. Its less error prone for the user